### PR TITLE
docs: update documentation for readme.md & phpdocs

### PR DESCRIPTION
**Pull request by maximgrynykha**
Hi, guys 👋

I had looked at the documentation using Grammarly twice. Then was fixed some spelling and typos according to it and lastly updated information about making screenshots.

(Currently the Grammarly with a free plan but it is better than nothing. In the future revision, I will make use of the paid plan.)

### CODE REVIEW

**Comment by enricodias** [comment]
Thanks for the contribution :)

Looks ok to me, but I'm not a native speaker. Let's wait for Graham to review it.

**Comment by GrahamCampbell** [comment]
Thanks for this. Could you please target 1.2, and make the same changes in the code. The README directly copies the phpdoc.

**Comment by maximgrynykha** [comment]
@GrahamCampbell  

If I correctly understood your remarks I need to target this pull-request to branch — 1.2 and copy the documentation of the full-page screenshot implementation [from my fork section](https://git.io/J1D0S) to the code in PHPDoc?

If so:

The method — [Page::getFullPageClip()](https://git.io/J1D0N) makes full-screen screenshots but not the full-page screenshot. Therefore, I'll update PHPDoc for this method. 

For the full-page screenshot documentation, I'll write in PHPDoc to the method — [Page::screenshot()](https://git.io/J1DEU) the same [description as in my fork section](https://git.io/J1D0S).

Finally, for the screenshot of the area on a page, I'll put the code [from my fork section](https://git.io/J1DEm) in PHPDoc to the method — [Page::screenshot()](https://git.io/J1DEU).

**Comment by GrahamCampbell** [comment]
Yes, please.

**Comment by maximgrynykha** [comment]
@GrahamCampbell 

Could you please apply fixes from StyleCI manually, or do I need to make this by myself with one more commit?

File: README.md [hunk 1]
```diff
@@ -591,15 +591,15 @@ if ($cookieBar) {
 }
 ```
 
-### Set user agent
+### Set User-Agent
```
Comment (GrahamCampbell) [inline README.md]: Please revert this title change.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (maximgrynykha) [reply README.md]: Thanks for noticing. I done for required changes.

**Pull request merged**